python/foglamp/tasks/north/sending_process.py

```diff
@@ -355,6 +359,23 @@ def __init__(self):
 
         self._event_loop = asyncio.get_event_loop()
 
+    @staticmethod
+    def _signal_handler(_signo, _stack_frame):
+        """ Handles signals to properly terminate the execution
+
+        Args:
+        Returns:
+        Raises:
+        """
+
+        SendingProcess._stop_execution = True
+
+        short_stack_frame = str(_stack_frame)[:100]
+        SendingProcess._logger.debug("{func} - signal captured |{signo}| - info |{ssf}| ".format(
```

Comment:
signo > signal_num?, can we avoid ssf,  purpose?

Reply:
done